src/account_manager.rs

```diff
@@ -677,7 +682,8 @@ mod tests {
                 .expect("invalid node URL")
                 .build();
 
-            crate::block_on(async move {
+            let mut runtime = tokio::runtime::Runtime::new().unwrap();
+            runtime.block_on(async move {
```

Comment:
This looks like a one time block so I assume it's fine

Reply:
this is just a test case; it will be a regular async test once I get the new stronghold up and running

Reply:
Whoops, I overlooked it, sorry